core/jobs.py

```diff
@@ -1012,7 +1012,7 @@ def validate(cls, unused_mapper_spec):
         Returns:
             bool. Whether mapper spec and all mapper patterns are valid.
         """
-        return True  # TODO.
+        return True  # TODO(seanlip).
```

Comment:
@seanlip, do we need a message here to explain the TODO?

Reply:
Good catch -- yes, we do.

@teddymarchildon could this be changed to "# TODO(seanlip): Actually implement the validation.` and put on the line above `return True`?